api/src/main/java/org/pmiops/workbench/api/CloudTaskInitialCreditsExpiryController.java

```diff
@@ -107,7 +107,7 @@ private void handleExpiredUsers(Set<DbUser> newlyExpiredUsers) {
           // delete apps and runtimes
           deleteAppsAndRuntimesInFreeTierWorkspaces(user);
           try {
-            mailService.alertUserInitialCreditsExpiration(user);
+            mailService.alertUserInitialCreditsExhausted(user);
```

Comment:
consider renaming this Controller too

Reply:
Good call. I made a ticket for that one: https://precisionmedicineinitiative.atlassian.net/browse/RW-13494